website/docs/reference/resource-configs/spark-configs.md

```diff
@@ -191,19 +191,22 @@ insert overwrite table analytics.spark_incremental
 
 
 **Usage notes:** The `merge` incremental strategy requires:
-- `file_format: delta`
-- Databricks Runtime 5.1 and above
+- `file_format: delta or hudi`
+- Databricks Runtime 5.1 and above for delta file format
+- Apache Spark for hudi file format
 
 dbt will run an [atomic `merge` statement](https://docs.databricks.com/spark/latest/spark-sql/language-manual/merge-into.html) which looks nearly identical to the default merge behavior on Snowflake and BigQuery. If a `unique_key` is specified (recommended), dbt will update old records with values from new records that match on the key column. If a `unique_key` is not specified, dbt will forgo match criteria and simply insert all new records (similar to `append` strategy).
 
 <Tabs
-  defaultValue="source"
+  defaultValue="delta_source"
```

Comment:
I understand why you added new tabs (and props for running with this docusaurus specific convention!). But since the difference is small, I propose generalizing the existing source and run code examples to accommodate both with the following changes:
1. for the source code tab:
    1. so in the source code mention both formats
        ```yml
        file_format='delta', # or 'hudi'
        ```
    2. change source file name to `file_incremental.sql` (or `lake_`? idk)
1. for the run code tab update the filename and model name to match whatever you decided previously

Reply:
Yes, even though I initially thought of it and then ignored it to be more precise, it makes sense to use single source code with hudi on comment since the rest of the code remains the same.